.changeset/cold-foxes-applaud.md

```diff
@@ -0,0 +1,13 @@
+---
+"@finos/legend-extension-mapping-generation": major
+"@finos/legend-graph": patch
+"@finos/legend-studio": patch
+---
+
+WHAT Add model loader extension for relational mapping generation, no breaking changes as the 
+extension is not added to studio yet
+WHY Mapping generation feature was made to allow users to merge their M2M mapping chain into 
+one relational mapping
+HOW consumer won't be able to leverage this change until new endpoint in engine is introduced 
+
+
```

Comment:
We don't need the other 2 changeset files. One is enough for this PR. also both legend-graph and legend-studio should ba a `minor` change